modules/signed-int16/programs/kdeapps.nix

```diff
@@ -0,0 +1,18 @@
+{
+  lib,
+  inputs,
+  config,
+  pkgs,
+  ...
+}: {
+  environment.systemPackages = with pkgs; [
+    kdePackages.merkuro
+    kdePackages.konversation
+    kdePackages.kdeconnect-kde
+    kdePackages.plasmatube
+  ];
```

Comment:
there's probably a better way to do this, please correct me if i'm wrong
something like 
```
kdePackages = [
]
```
should be a thing, but i tried and failed